### PR TITLE
Add docker-compose documentation

### PR DESCRIPTION
**Pull request by pascalgrimaud**
Refer to https://github.com/jhipster/generator-jhipster/pull/2032
Don't hesitate to review, change or correct my formulation.

### CODE REVIEW

File: creating_an_app.html [hunk 1]
```diff
@@ -126,3 +126,76 @@
...
+<b>WARNING!</b> This is a new feature. Only these containers are available : MySQL, PostgreSQL, MongoDB, Cassandra
+
+<h3>Description</h3>
+<p>
+  At these questions :
+  <ul>
+    <li>"Which *production* database would you like to use?"</li>
+    <li>"Which" *development* database would you like to use?"</li>
+  </ul>
+  if you choose MySQL, PostgreSQL, MongoDB or Cassandra, a <code>docker-compose.yml</code> or <code>docker-compose-prod.yml</code> files are generated in your folder project.<br>So you can use docker-compose to start your database in development or production profile.</p>
```
Comment (jdubois) [inline creating_an_app.html]: wording: "files are generated" -> "file is generated"

**Comment by atomfrede** [comment]
I would add a simple one liner how to stop (and maybe remove) containers

**Comment by jdubois** [comment]
Looks good, but I'm going to move this to a separate page, it deserves its own page!!

**Comment by pascalgrimaud** [comment]
@atomfrede : good idea to add some basic docker commands (list, stop and remove container)
Try to do it tonight

**Pull request merged**